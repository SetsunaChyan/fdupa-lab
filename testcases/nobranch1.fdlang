function main(){
x = input();
y = input();

x = x + y;

check_interval(x, 0, 255);
check_interval(y, 0, 255);

x = x + 100;
y = y - 100;

check_interval(x, 100, 255);
check_interval(y, 0, 155);

x = x + 100;
y = y - 100;

check_interval(x, 200, 255);
check_interval(y, 0, 55);

x = x + 100;
y = y - 100;

check_interval(x, 255, 255);
check_interval(y, 0, 0);
}