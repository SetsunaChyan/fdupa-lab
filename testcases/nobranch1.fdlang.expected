Line 7: YES; x in [0, 255]
Line 8: YES; y in [0, 255]
Line 13: YES; x in [100, 255]
Line 14: YES; y in [0, 155]
Line 19: YES; x in [200, 255]
Line 20: YES; y in [0, 55]
Line 25: YES; x in [255, 255]
Line 26: YES; y in [0, 0]
